src/components/TimePicker/TimePicker.jsx

```diff
@@ -194,15 +204,17 @@ class TimePicker extends React.Component<Props, State> {
     } = this.props;
     switch (defaultOption) {
       case TimePicker.DefaultOptions.nextInterval: {
-        const roundedUp = Math.ceil(moment().minute() / timeIntervalMinutes) * timeIntervalMinutes;
+        const now = new Date();
+        const roundedUp = Math.ceil(now.getMinutes() / timeIntervalMinutes) * timeIntervalMinutes;
 
-        return moment()
-          .minute(roundedUp)
-          .second(0)
-          .format(TimePicker.FormatTypes.timeFormat24);
+        const defaultTime = new Date(now);
+        defaultTime.setMinutes(roundedUp);
```

Comment:
can be chained?

Reply:
Nope. I tried :-( 

it returns the new minutes as an int